scripts/debug-fb-www.js

```diff
@@ -194,8 +194,6 @@ readComponentsList()
 
     let timeTaken = Math.floor((Date.now() - startTime) / 1000);
     console.log(`${chalk.gray(`Compile time`)}: ${timeTaken}s\n`);
-    // warning about ref and keys
-    console.log(`Warning: the build assumes that ref and key aren't being spread.`);
```

Comment:
Why do we think it's safe to remove this? Maybe it is but only for the first render?

Here's a test case with spread ref that fails on master and still fails on this PR:

```js
var React = require('react');
this['React'] = React;

class Wat extends React.Component {
  render() {
    return <div {...this.props} />;
  }
}

function App(props) {
  return <Wat {...props.inner} />;
}

App.getTrials = function(renderer, Root) {
  var obj;
  function ref(inst) {
    obj = inst;
  }
  renderer.update(<Root inner={{className: 'foo', ref}} />);
  return [
    ['simple render with jsx spread 6', renderer.toJSON()],
    ['type', Object.keys(obj).join(',')]
  ];
};

if (this.__optimizeReactComponentTree) {
  __optimizeReactComponentTree(App);
}

module.exports = App;
```

Reply:
Oops, I wasn't meant to remove that. Nice spot.

Reply:
Test added :)